src/fu-tool.c

```diff
@@ -636,6 +637,9 @@ main (int argc, char *argv[])
 		{ "show-all-devices", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all_devices,
 			/* TRANSLATORS: command line option */
 			_("Show devices that are not updatable"), NULL },
+		{ "plugin-whitelist", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &plugin_glob,
```

Comment:
You'll want to add this to the new fwupdtool bash completion.

Reply:
I'm not sure I know how; you can specify the plugin-whitelist option multiple times. Help most welcome! :)

Reply:
I think this should be fine actually:

```
--- a/data/bash-completion/fwupdtool
+++ b/data/bash-completion/fwupdtool
@@ -15,6 +15,7 @@ _fwupdmgr_opts=(
        '--allow-older'
        '--force'
        '--show-all-devices'
+       '--plugin-whitelist'
 )
```

That will show it multiple times if you tab over and over.  I think autocompleting the plugins supported is going to be difficult from bash completion.

Given the type of use you envision from this I think that should be OK though :)

Reply:
Okay, I'll do that. I've also added an additional `get-plugins` command to fwupdtool if that helps with bash completion.

Reply:
Alright I added a commit to your branch that should make it work.